src/Render/Program.cs

```diff
@@ -67,17 +67,22 @@ internal static class Program {
 			var links = doc.Descendants().OfType<LinkInline>();
                         
 			foreach( var link in links ) {
-				if ( link.Url.EndsWith( ".md" ) ) {
+				string url = link.GetDynamicUrl?.Invoke() ?? link.Url;

```

Comment:
What's GetDynamicUrl about?

Reply:
Apparently it's a function that gets the URL sometimes? We use it in `D2LLinkInlineRenderer` too. I'm not sure it is ever actually used, but I put it as a just in case.